components/page_info/BUILD.gn

```diff
@@ -4,5 +4,8 @@
 # You can obtain one at http://mozilla.org/MPL/2.0/.
 
 group("page_info") {
-  deps = [ "//brave/components/vector_icons" ]
+  deps = [
+    "//brave/components/vector_icons",
+    "//brave/components/ipfs"
```

Comment:
Sorry, forgot to mention: also, I think, the suggested practice is to add deps for all includes directly, which also means adding a dep on `//brave/components/ipfs/buildflags`. So, maybe add
`import("//brave/components/ipfs/buildflags/buildflags.gni")`
and `//brave/components/ipfs/buildflags` to `deps`, WDYT?

Reply:
Correction: importing `//brave/components/ipfs/buildflags/buildflags.gni` isn't needed.

Reply:
Or rather, maybe:
<pre>
import("//brave/components/ipfs/buildflags/buildflags.gni")

group("page_info") {
  deps = [
  "//brave/components/ipfs/buildflags",
   "//brave/components/vector_icons",
 ]

  if (ipfs_enabled) {
    deps += [ "//brave/components/ipfs" ]
  }
}
</pre>